Makefile

```diff
@@ -56,3 +60,16 @@ sync: install
 
 update-blocklist-file:
 	wget -O blocklist.xml $(BLOCKLIST_FILE_URL)
+
+update-schemas:
+	wget -O schemas.json $(AMO_BLOCKLIST_UI_SCHEMA)
+
+install-kinto: $(VENV)/bin/kinto
+$(VENV)/bin/kinto: install
+	$(VENV)/bin/pip install kinto
+
+run-kinto: $(VENV)/bin/kinto
+	$(VENV)/bin/kinto --ini config/kinto.ini start
+
+need-kinto-running:
+	@curl http://localhost:8888/v1 2>/dev/null 1>&2 || (echo "Run 'make run-signer' before starting tests." && exit 1)
```

Comment:
Can you explain what this line is doing? It's confusing, is the "echo" relevant?